package.json

```diff
@@ -29,24 +29,24 @@
   },
   "homepage": "https://github.com/Unibeautify/beautifier-prettier#readme",
   "peerDependencies": {
-    "unibeautify": ">= 0.11.1",
-    "prettier": "^1.11.1"
+    "unibeautify": ">= 0.12.1",
+    "prettier": "^1.12.1"
```

Comment:
Unless there is a new feature which is explicitly required, we do not need to increment the `peerDependencies`.

The `peerDependencies` version should be the lowest possible version which will work successfully.